strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -15,43 +15,42 @@
 """Bosonic backend"""
 import warnings
 
-from numpy import (
-    empty,
-    concatenate,
-    arange,
-    array,
-    identity,
-    arctan2,
-    angle,
-    sqrt,
-    vstack,
-    zeros_like,
-    allclose,
-    ix_,
-    zeros,
-    shape,
-)
+import numpy as np
+from scipy.linalg import block_diag
+
 from thewalrus.samples import hafnian_sample_state, torontonian_sample_state
 import itertools as it
 
 from strawberryfields.backends import BaseBosonic
 from strawberryfields.backends.shared_ops import changebasis
 from strawberryfields.backends.states import BaseBosonicState
+import strawberryfields.ops as ops
```

Comment:
wonder if this is causing trouble, maybe import only the required ops functions/methods `Bosonic`, `Catstate`, `Comb` and `Fock`.

Reply:
Done